my-app/src/pages/app.js

```diff
@@ -0,0 +1,16 @@
+import React, {Component} from 'react';
```

Comment:
Replace app component to root folder.
Also, you didn't use default export, but import is like you export by default. U need to use {module-name}. And you don't need to write so deep path to module since you used index.js. What u need is `import {Pdp} from './pdp';`

>

Reply:
Do all of your suggests.